src/LinAlg/hiopVectorPar.cpp

```diff
@@ -120,11 +121,15 @@ hiopVector* hiopVectorPar::new_copy () const
 
 void hiopVectorPar::setToZero()
 {
-  for(int i=0; i<n_local_; i++) data_[i]=0.0;
+  for(int i=0; i<n_local_; i++) {
+    data_[i]=0.0;
+  }
```

Comment:
I am fine with the code, but style guide implies that we should add spaces before and after '=' and '<'. \
There are many places where we do not add spaces before and after `=`, especially in the logical conditions for the loops. Should we update the style guide?

Reply:
I don't remember (and I can't find in the style guidelines) to be specific about those spaces. It can go either way afaik, but I tend to not use them.